### ISSUE TITLE
Cannot expose the property twice

### ISSUE THREAD

**Issue by faucct** [ruby-grape/grape-entity#175]
It is probably intended to behave another way.
```ruby
class ObjectSerializer < Grape::Entity
  expose(:iso2, as: :id)
  expose(:iso2)
end

ObjectSerializer.new(object).as_json
# => { iso2: 1 }
```

**Comment by marshall-lee** [comment]
I believe it's already fixed in current `master` (0.5.0) after https://github.com/ruby-grape/grape-entity/pull/151 where `exposures` hash table is substituted with array. But `0-4` branch won't be fixed, sorry. It's impossible to backport #151 without breaking changes.

You could try `master` in new project but in existing project relying on 0.4.* there can be troubles when upgrading to 0.5.0 (sorry, i've not written `UPGRADING` document yet).

If it still doesn't work as expected please contribute with PR with failing test.

**Comment by faucct** [comment]
Okay, thanks.
